rlinf/utils/data_iter_utils.py

```diff
@@ -567,6 +564,34 @@ def get_iterator_dynamic(
     return itertools.chain(microbatches), partitions, n_micro_batch
 
 
+def split_dynamic_batch_size(
+    batch: Dict[str, torch.Tensor],
+    cp_world_size: int,
+    vpp_world_size: int,
+    max_tokens_per_mbs: int,
+    microbatch_group_size_per_vp_stage: int,
+):
+    """Split a global batch using dynamic batch sizing."""
+    max_tokens_per_mbs = max_tokens_per_mbs * cp_world_size
+    vpp_size = vpp_world_size
+    if vpp_size is not None and vpp_size > 1:
+        microbatch_group_size_per_vp_stage = microbatch_group_size_per_vp_stage
```

Comment:
what is this line used for?

Reply:
I'm not sure. I just move the dynamic batch size code here.